### ISSUE TITLE
Detect when str method is called as property

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#865]
# Rule request

## Thesis

Some str methods look like properties, but they are not. I can easily remain cases when I forgot to call them.

Bad:

```python
'a'.islower
'a'.upper
```

Good:

```python
'a'.islower()
'a'.upper()
```

It's cool also resolve simple variables:

```python
a = 'text'
a.isupper
```

## Reasoning

Runtime errors make me sad. Especially when it's `if 'a'.isupper: ...` (that's always truely).

**Comment by sobolevn** [comment]
Sorry, but that's `mypy` job. From our perspective these a valid calls in several cases. It only makes sense when you know the context (and we don't know the context).
